examples/synthetic_records.ipynb

```diff
@@ -41,12 +41,12 @@
     "from gretel_synthetics.config import LocalConfig\n",
     "\n",
     "# Create a config that we can use for both training and generating, with CPU-friendly settings\n",
-    "# The default values for ``max_chars`` and ``epochs`` are better suited for GPUs\n",
+    "# The default values for ``max_lines`` and ``epochs`` are better suited for GPUs\n",
     "\n",
     "config = LocalConfig(\n",
-    "    max_chars=100000,  # friendly towards CPUs\n",
+    "    max_lines=0,  # friendly towards CPUs\n",
```

Comment:
There's a mismatch here. This is also the default value, which per the above comment is better suited for GPUs and hence not CPU friendly. This should thus be a value other than `0`, or entirely removed, along with its mention in the comment 3 lines above.

Reply:
nice catch- updated to a sane number for training on CPU (5000 lines)